src/test/developTest/resources/wdr/wdr_video_details1.html

```diff
@@ -0,0 +1,976 @@
+<div class="duration">28:14</div>
+</div>
+</div>
+<span>Rudis Leben für die Schlittenhunde</span> <span>|</span> 09.01.2017 <span>|</span> <strong>video</strong>
+</a>
+</li>
+<li >
+<a href="/mediathek/video/sendungen/stratmann-wandert/video-stratmann-wandert-100.html" data-extension='{"mediaObj":{"offset":"0.0"}}' title="Stratmann wandert, WDR">
+<div class="mediaCon">
+<div class="media mediaA">
+<img data-ctrl-attributeswap="{&#034;id&#034;:&#034;tva-wdr-stratmann-wandert-weihnachten-100&#034;,&#034;action&#034;:{&#034;xs&#034;:{&#034;src&#034;:&#034;/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg&#034;},&#034;s&#034;:{&#034;src&#034;:&#034;/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg&#034;},&#034;m&#034;:{&#034;src&#034;:&#034;/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg&#034;},&#034;l&#034;:{&#034;src&#034;:&#034;/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg&#034;},&#034;xl&#034;:{&#034;src&#034;:&#034;/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg&#034;}}}" class="img" alt="Dr. Ludger Stratmann und sabine heinrich wandern im rothhhaargebirge" title="Dr. Ludger Stratmann und sabine heinrich wandern im rothhhaargebirge | Bildquelle: wdr" src="/tva-wdr-stratmann-wandert-weihnachten-100~_v-ARDKleinerTeaser.jpg"/>
+<div class="duration">43:59</div>
+</div>
+</div>
+<span>Stratmann wandert</span> <span>|</span> 26.12.2016 <span>|</span> <strong>video</strong>
+</a>
+</li>
+</ul>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div class="con">
+<h2 class="conHeadline hidden">Neuer Abschnitt</h2>
+<div class="modCon">
+<div class="mod modA">
+<div aria-live="polite" class="boxCon">
+<div class="box" data-ctrl-dummy-entry="{}">
+<h3 class="ressort">Abenteuer Erde</h3>
+<div class="teaser">
+<div class="linklist">
+<ul id='relatedContent' class="list">
+<li >
+<a href="/mediathek/video/sendungen/abenteuer-erde/index.html" >
+<span>Alle Videos der Sendung</span> <span>|</span> <strong>mehr</strong>
+</a>
+</li>
+</ul>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div class="con">
+<h2 class="conHeadline hidden">Neuer Abschnitt</h2>
+<div class="modCon">
+<div class="mod modA">
+<div aria-live="polite" class="boxCon">
+<div class="box" data-ctrl-dummy-entry="{}">
+<h3 class="ressort">Startseite Abenteuer Erde</h3>
+<div class="teaser">
+<div class="linklist">
+<ul id='relatedContent' class="list">
+<li >
+<a href="/fernsehen/abenteuer-erde/index.html" >
+<span>Abenteuer Erde - Infos zur Sendung</span> <span>|</span> <strong>mehr</strong>
+</a>
+</li>
+</ul>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div class="section sectionA" >
+<div class="con">
+<div class="socialMedia">
+<div data-ctrl-sharingmedia="" class="mod modSharing">
+<div class="fixedSharingContainer">
+<div class="toggleShare">
+<img src="/resources/img/base/icon/sharing_toggle.svg">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AImgWithoutAltCheck)